api/hasura/cron/dailyUpdate.ts

```diff
@@ -249,6 +254,44 @@ async function handler(req: VercelRequest, res: VercelResponse) {
             );
         }
       }
+
+      if (circle.discord_circle) {
+        const {
+          discord_channel_id: channelId,
+          discord_role_id: roleId,
+          alerts,
+        } = circle?.discord_circle || {};
```

Comment:
```suggestion
        } = circle.discord_circle || {};
```

shouldn't be necessary since it's in the if-clause